rascaline/src/calculators/lode/radial_integral/spline.rs

```diff
@@ -24,7 +24,7 @@ pub struct LodeRadialIntegralSplineParameters {
     pub max_radial: usize,
     /// Number of angular components
     pub max_angular: usize,
-    /// cutoff radius, this is also the maximal value that can be interpolated
+    /// k-space cutoff radius, this is also the maximal value that can be interpolated
     pub cutoff: f64,
```

Comment:
Maybe we should call this also this `k_cutoff` to avoid confusion with the real space version?

Reply:
yes, that sounds like a good idea!